### ISSUE TITLE
How to add static files in the project to the output html folder?

### ISSUE THREAD

**Issue by Thaina** [intellifactory/websharper#710]
I want to use websharper html client only. And I think I could easily add static file into output html folder

Where is the project setting or include script I need to set to make it automatically build static content to output folder?

**Comment by Tarmil** [comment]
You can list your static files in the `extra.files` file in your project.

**Comment by Thaina** [comment]
What is `extra.files` ? Is it folder name?

**Comment by Thaina** [comment]
@Tarmil Thank you very much

I use Zafir HTML project and it doesn't make that file when project was created

**Comment by Tarmil** [comment]
Right, if it's called "Zafir" then you have an older version of the VS extension. What exact version of the extension do you have? Can you please try the latest extension? Note that templates are now called "WebSharper 4 XYZ" instead of "Zafir XYZ".
